docs/semgrep-ci/configuration-reference.md

```diff
@@ -115,14 +115,14 @@ export SEMGREP_RULES="p/default no-exec.yml"
 Do not set `SEMGREP_APP_TOKEN` environment variable within the same CI job as `SEMGREP_RULES`.
 :::
 
-### `SEMGREP_TIMEOUT`
+### `DEFAULT_TIMEOUT`
 
-Set `SEMGREP_TIMEOUT` to define a custom timeout. The value must be in seconds. The default value is 1800 seconds (30 minutes).
+Set `DEFAULT_TIMEOUT` to define a custom timeout. The value must be in seconds. The default value is 30 seconds. This timeout refers to the maximum amount of time Semgrep will spend scanning a single file. Dy default, it will attempt to scan each file with this timeout three times; you can control this using `--timeout-threshold`.
```

Comment:
```suggestion
Set `DEFAULT_TIMEOUT` to define a custom timeout. The value must be in seconds. The default value is 30 seconds. This timeout refers to the maximum amount of time Semgrep will spend scanning a single file. By default, it will attempt to scan each file with this timeout three times; you can control this using `--timeout-threshold`.
```

Reply:
Building on the previous suggestion. Do not use future tense in docs unless you are referring to specific event in future (see: https://www.notion.so/r2cdev/r2c-technical-communication-style-guide-4d1104e78927460aa8c7624aedd6561f#ca0bfefb2cfb42339cacc8eeb8a3554f) @holdenmcg 

```suggestion
Set `DEFAULT_TIMEOUT` to define a custom timeout. The value must be in seconds. The default value is 30 seconds. This timeout refers to the maximum amount of time Semgrep spends scanning a single file. By default, it attempts to scan each file with this timeout three times; you can control this using `--timeout-threshold`.
```